ATM_Selenium_Training/src/test/java/com/epam/atm/selenium/NewApplicationTest.java

```diff
@@ -0,0 +1,74 @@
+package com.epam.atm.selenium;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class NewApplicationTest extends BaseTest {
+
+    @Test
+    public void createNewApplicationWithDefaultSetup() {
+        int x = (int) (Math.random() * 1000);
+        driver.get("https://network-international--qa.cs83.my.salesforce.com/");
+        System.out.println("Open application (url) for testing");
+        System.out.println("Check that correct Login page is opened");
+        Assert.assertEquals(driver.getTitle(), "Login | Salesforce", "Incorrect page (login page) is opened");
+        WebElement usernameInput = driver.findElementByXPath("//input[@class='input r4 wide mb16 mt8 username']");
+        usernameInput.sendKeys("yauhen_valodzin@epam.com.qa");
+        System.out.println("Find and populate Username field");
+        WebElement passwordInput = driver.findElementByXPath("//input[@class='input r4 wide mb16 mt8 password']");
+        passwordInput.sendKeys("Ceakt_1234");
+        System.out.println("Find and populate Password field");
+        WebElement loginButton = driver.findElementByXPath("//input[@class='button r4 wide primary']");
+        loginButton.click();
+        System.out.println("Find and click Login button");
+        System.out.println("Add explicit wait");
+        new WebDriverWait(driver, 10).until(ExpectedConditions.presenceOfElementLocated((By.xpath("//span[text()='Recently Viewed']"))));
+        System.out.println("Check that correct Home page (after login) is opened");
+        Assert.assertEquals(driver.getTitle(), "Recently Viewed | Applications | Salesforce", "Incorrect Home page is not opened");
```

Comment:
Move expected text results to constants also

Reply:
Done